_posts/2022-01-27-release-statefun-3.2.0.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title:  "Stateful Functions 3.2.0 Release Announcement"
+subtitle: "The Apache Flink community is happy to announce the release of Stateful Functions (StateFun) 3.2.0."
+date: 2022-01-27T08:00:00.000Z
+categories: news
+authors:
+- trohrmann:
+  name: "Till Rohrmann"
+  twitter: "stsffap"
+- igalshilman:
+  name: "Igal Shilman"
+  twitter: "IgalShilman"
+---
+
+Stateful Functions is a cross-platform stack for building Stateful Serverless applications, making it radically simpler to develop scalable, consistent, and elastic distributed applications.
+This new release brings various improvements to the StateFun runtime, a leaner way to specify StateFun module components, and a brand new JavaScript SDK!
+
+The binary distribution and source artifacts are now available on the updated [Downloads](https://flink.apache.org/downloads.html)
+page of the Flink website, and the most recent Java SDK, Python SDK,, GoLang SDK and JavaScript SDK distributions are available on [Maven](https://search.maven.org/artifact/org.apache.flink/statefun-sdk-java/3.2.0/jar), [PyPI](https://pypi.org/project/apache-flink-statefun/), [Github](https://github.com/apache/flink-statefun/tree/statefun-sdk-go/v3.2.0), and [npm](https://www.npmjs.com/package/apache-flink-statefun) respectively.
+You can also find official StateFun Docker images of the new version on [Dockerhub](https://hub.docker.com/r/apache/flink-statefun).
+
+For more details, check the complete [release changelog](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12350540)
+and the [updated documentation]({{site.DOCS_BASE_URL}}flink-statefun-docs-release-3.2/).
+We encourage you to download the release and share your feedback with the community through the [Flink mailing lists](https://flink.apache.org/community.html#mailing-lists)
+or [JIRA](https://issues.apache.org/jira/browse/)!
+
+{% toc %}
+
+## New Features
+
+### A brand new JavaScript SDK
```

Comment:
Perhaps we should say that this is for NodeJs in particular? 
Something like
`A brand new JavaScript SDK for NodeJS`

Reply:
This is better. I will change it.